packages/url_launcher/url_launcher/lib/url_launcher.dart

```diff
@@ -44,6 +44,9 @@ import 'package:url_launcher_platform_interface/url_launcher_platform_interface.
 /// [enableDomStorage] is an Android only setting. If true, WebView enable
 /// DOM storage.
 /// [headers] is an Android only setting that adds headers to the WebView.
+/// Adding headers without using a WebView is not fully supported.
```

Comment:
I think that a reader of this comment is looking to understand whether and under what circumstances he can use this. Maybe something like:

```
When not using a WebView, the header information is passed to the browser, some Android browsers do not support the [Browser.EXTRA_HEADERS](https://developer.android.com/reference/android/provider/Browser#EXTRA_HEADERS) intent extra and will lose the header information.
```